providers/linux/testdata/fedora30/proc/stat

```diff
@@ -0,0 +1,12 @@
+cpu  31608779 223223 15924288 1134746753 223880 4348430 1995355 266903 0 0
```

Comment:
I don't see where this file is being used in the PR. If it's not used then let's remove it.

Reply:
Apparently the code will try and find a `/proc/host` file before it does anything else. Without this the test will fail.